r88_airsim_params.parm

```diff
@@ -21,15 +21,39 @@ ATC_RAT_YAW_D	0.09
 ATC_RAT_YAW_I	0.2
 ATC_RAT_YAW_P	2.0
 
+# Position control params
+PSC_JERK_XY      10.0
```

Comment:
If we want to edit this in the field, can we connect it to a launch arg, or would we edit through QGC?

Reply:
Jk this is just for sim, that's fine then!